pkg/scheduler/ugm/user_tracker.go

```diff
@@ -97,3 +97,7 @@ func (ut *UserTracker) GetUserResourceUsageDAOInfo() *dao.UserResourceUsageDAOIn
 	userResourceUsage.Queues = ut.queueTracker.getResourceUsageDAOInfo("")
 	return userResourceUsage
 }
+
+func (ut *UserTracker) CleanDeletedQueueTracker(queuePath string) {
```

Comment:
Same here, `DeleteQueueTracker()`

Reply:
Good suggestion, addressed in latest PR.